code/game/gamemodes/miniantags/abduction/abduction.dm

```diff
@@ -149,32 +149,36 @@
 	var/datum/objective/stay_hidden/O = new
 	abductor.objectives += O
 	abductor.objectives += team_objectives[team_number]
 	var/team_name = team_names[team_number]
 
 	SEND_SOUND(abductor.current, sound('sound/ambience/antag/abductors.ogg'))
-	to_chat(abductor.current, "<span class='notice'>You are an agent of [team_name]!</span>")
-	to_chat(abductor.current, "<span class='notice'>With the help of your teammate, kidnap and experiment on station crew members!</span>")
-	to_chat(abductor.current, "<span class='notice'>Use your stealth technology and equipment to incapacitate humans for your scientist to retrieve.</span>")
-	to_chat(abductor.current, "<span class='motd'>For more information, check the wiki page: ([GLOB.configuration.url.wiki_url]/index.php/Abductor)</span>")
+	var/list/messages = list()
+	messages.Add("<span class='notice'>You are an agent of [team_name]!</span>")
+	messages.Add("<span class='notice'>With the help of your teammate, kidnap and experiment on station crew members!</span>")
+	messages.Add("<span class='notice'>Use your stealth technology and equipment to incapacitate humans for your scientist to retrieve.</span>")
+	messages.Add("<span class='motd'>For more information, check the wiki page: ([GLOB.configuration.url.wiki_url]/index.php/Abductor)</span>")
+	to_chat(abductor.current, chat_box_red(messages.Join("<br>")))
 	abductor.current.create_log(MISC_LOG, "[abductor.current] was made into an abductor agent")
 
 	abductor.announce_objectives()
 
-/datum/game_mode/abduction/proc/greet_scientist(datum/mind/abductor,team_number)
+/datum/game_mode/abduction/proc/greet_scientist(datum/mind/abductor, team_number)
 	var/datum/objective/stay_hidden/O = new
 	abductor.objectives += O
 	abductor.objectives += team_objectives[team_number]
 	var/team_name = team_names[team_number]
 
 	SEND_SOUND(abductor.current, sound('sound/ambience/antag/abductors.ogg'))
-	to_chat(abductor.current, "<span class='notice'>You are a scientist of [team_name]!</span>")
-	to_chat(abductor.current, "<span class='notice'>With the help of your teammate, kidnap and experiment on station crew members!</span>")
-	to_chat(abductor.current, "<span class='notice'>Use your tool and ship consoles to support the agent and retrieve human specimens.</span>")
-	to_chat(abductor.current, "<span class='motd'>For more information, check the wiki page: ([GLOB.configuration.url.wiki_url]/index.php/Abductor)</span>")
+	var/list/messages = list()
+	messages.Add("<span class='notice'>You are a scientist of [team_name]!</span>")
+	messages.Add("<span class='notice'>With the help of your teammate, kidnap and experiment on station crew members!</span>")
+	messages.Add("<span class='notice'>Use your tool and ship consoles to support the agent and retrieve human specimens.</span>")
+	messages.Add("<span class='motd'>For more information, check the wiki page: ([GLOB.configuration.url.wiki_url]/index.php/Abductor)</span>")
```

Comment:
```suggestion
	messages.Add("<span class='motd'>For more information, check the wiki page: [wiki_link("Abductor", "Abductor")]</span>")
```
Same as above. Though I would test it, as I never have used it myself before since it is new.